modules/post/osx/gather/terminal_savedstate.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -
+# Only works on python3
+
+from metasploit import module
+import logging
+
+import re
+import sys
+import json
+import struct
+import binascii
+import collections
+import cryptography.hazmat.backends
+import cryptography.hazmat.primitives.ciphers
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+# this may need to move inside run so it doesn't crash all of msf if its not on the system
+try:
+    import hexdump
+except ImportError:
+    print('[e] Please install `hexdump` via pip')
+    sys.exit(-1)
+
+from lib.msf.post.osx import bplist
+
+metadata = {
+    'name': 'OSX Terminal Saved State File Recovery',
+    'description': '''
+        This module enumerates the saved state files for the terminal
+        application on OSX.  These files are AES-128-CBC encrypted but
+        we're able to pull the key out as well for decryption.
+        The files themselves contain an exact copy of what was sent
+        to and from the terminal, which may include sensitive information.
+    ''',
+    'authors': [
+        'h00die', # msf module
+        'Willi Ballenthin <willi.ballenthin@gmail.com>', # PoC
+        'kshitij Kumar <kshitij.kumar@crowdstrike.com>' # PoC
+    ],
+    'date': '2019-10-01',
+    'license': '<license>',
```

Comment:
remove/fix

Reply:
This broke loading. I changed it to `BSD_LICENSE` as per https://github.com/CrowdStrike/automactc/blob/master/LICENSE. I didn't see a license for https://gist.github.com/williballenthin/ab23abd5eec5bf5a272bfcfb2342ec04.

Reply:
https://gist.github.com/williballenthin/994db929b1448fdf73baf91207129dec#file-macos_savedstate-py-L5